### ISSUE TITLE
Cannot find module applicationinsights

### ISSUE THREAD

**Issue by pelikhan** [Microsoft/AppInsights-node.js#5]
After running npm install applicationinsights, I get the following error:

* test.js
````
var ai = require('applicationinsights')
````

````
>node test.js
module.js:338
    throw err;
          ^
Error: Cannot find module 'applicationinsights'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (C:\cb\td\win8\temp\test.js:1:72)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
````

**Comment by lukehoban** [comment]
I ran into this today as well.  It appears that the repo and NPM package currently only provide the `.ts` files, not the compiled `.js` files.  That means that the package can only be used directly by TypeScript consumers right now.

I just added a pull request https://github.com/Microsoft/AppInsights-node.js/pull/6 which fixes this, and also makes some changes for more idiomatic Node.js and TypeScript usage.
